kuma/attachments/models.py

```diff
@@ -49,6 +72,9 @@ def __str__(self):
         return self.title
 
     def get_file_url(self):
+        if self.current_revision is None:
+            return ""
```

Comment:
I know you didn't write the `on_delete=models.SET_NULL` on line 52 but doesn't that seem very odd?
How can a file attachment not belong to a revision? Won't it be unfreed spaces that's floating without an owner? I.e. bad in Rust. 

Arguably, returning an empty string won't cause an exception but I'd favor something like:
```python
if not self.current_revision:
    raise ValueError("Attachments without a revision don't have a file URL")
```

This won't necessarily need a unit test but whatever code that calls and depends on `get_file_url` should be aware of this and deal with it accordingly. I think that'd be a lot crisper than dealing with the surprising effect of a URL that is an empty string. Because an empty string definitely isn't a "URL".

Reply:
Will file a separate bug to track resolving this. The need to file that bug is captured in the post-merge checklist at the top of this issue.